tests/test_addons.py

```diff
@@ -217,7 +217,8 @@ def build_payload(self, metadata, **kwargs):
             'signature': signature,
         }
 
-    def test_add_log(self):
+    @mock.patch('website.addons.base.notifications.file_notify')
```

Comment:
If we're mocking `file_notify`, shouldn't we also assert that it was called?

I get that this is testing that a log was added, but I think I'd rather the test fail when the log functionality is broken than later have the mock be unnecessary and the test pass.